### DISCUSSION TITLE
byedpi  and kodi

### DISCUSSION THREAD

**Discussion by edvejak** [hufrea/byedpi#62] [General]
Hi. How to use byedpi  in kodi?

**Comment by edvejak** [comment]
Whether I need to start proxy for kodi or something else?

**Comment by lufog** [comment]
Start proxy (run ciadpi) and set it in settings (Settings/System/Internet access/Use proxy server). 
Proxy type: SOKCS5
Server: 127.0.0.1 (unless changed with `-i`)
Port: 1080 (unless changed with `-p`)

By the way, I don’t use Kodi myself, and I don’t know if it’s possible to run external programs in it.

**Comment by edvejak** [comment]
> Start proxy (run ciadpi) and set it in settings (Settings/System/Internet access/Use proxy server). Proxy type: SOKCS5 Server: 127.0.0.1 (unless changed with `-i`) Port: 1080 (unless changed with `-p`)
> 
> By the way, I don’t use Kodi myself, and I don’t know if it’s possible to run external programs in it.

After I made this I have endless spamming massage: errno 98 adress already in use. I tried 1080, 10801. I tried it in kodi preferences and in youtube plugin preferences with the same negative result.
